### ISSUE TITLE
errors after starting `main.py`

### ISSUE THREAD

**Issue by mb6ockatf** [IvanIsak2000/py-run-screenshot#2]
here's traceback:
```python3
$ ./main.py

./main.py: line 5: from: command not found
./main.py: line 9: syntax error near unexpected token `('
./main.py: line 9: `def main():'
```

note that line 2: import time - was never used

how may i fix it?

UPD. it must be something platform-dependent (i have no windows machine rn, so could not test it)

**Comment by IvanIsak2000** [comment]
First of all, make sure that this screenshot fragment will be understandable to the program, for now I recommend running programs that do not have quotes:

Recommended :
print(111111113212323)

![image](https://user-images.githubusercontent.com/79650307/229306422-7a48174d-a037-4e1d-88a1-95e5731617e6.png)



So far, poorly recognized:
print('Hello world')

![image](https://user-images.githubusercontent.com/79650307/229306440-ff4413b9-43a8-41f1-af37-e811405db6e7.png)
`The program does not correctly recognize quotes`

**Comment by IvanIsak2000** [comment]
> here's traceback:
> 
> ```python
> $ ./main.py
> 
> ./main.py: line 5: from: command not found
> ./main.py: line 9: syntax error near unexpected token `('
> ./main.py: line 9: `def main():'
> ```
> 
> note that line 2: import time - was never used
> 
> how may i fix it?
> 
> UPD. it must be something platform-dependent (i have no windows machine rn, so could not test it)

Can you show your screenshot?

**Comment by mb6ockatf** [comment]
> > here's traceback:
> > ```python
> > $ ./main.py
> > 
> > ./main.py: line 5: from: command not found
> > ./main.py: line 9: syntax error near unexpected token `('
> > ./main.py: line 9: `def main():'
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > note that line 2: import time - was never used
> > how may i fix it?
> > UPD. it must be something platform-dependent (i have no windows machine rn, so could not test it)
> 
> Can you show your screenshot?

@IvanIsak2000 it's all about #3 . the program couldn't grab my screenshot (and consequently, distinguish any symbols there)

**Comment by mb6ockatf** [comment]
Closed as duplicate of #3
